site/docs/types.md

```diff
@@ -29,13 +29,25 @@ Reference documentation for common parameter **types** expected by Vega-Lite spe
 
 Accepts any literal value, including a string, number, boolean, or `null`.
 
+<a name="align" href="#align">#</a>
+**Align**
+
+Possible values: `"left"`, `"center"`, or `"right"`.
+
 <br/><a name="array" href="#array">#</a>
 **Array** or **Type[]**
 
 Accepts array values. For example: `[]`, `[1, 2, 3]`, `["foo", "bar"]`. If individual array items must adhere to a specific type, bracket notation &ndash; such as `Number[]` or `String[]` &ndash; is used to indicate the item type.
 
 In most cases, arrays may also have [signal references](#Signal) as items. For example: `[{"signal": "width"}, {"signal": "height"}]`.
 
+<a name="baseline" href="#baseline">#</a>
+**Baseline**
+
+How text such as legends, labels, or titles should be aligned vertically.
+
+Possible values: `"top"`, `"middle"`, `"bottom"`, or `"alphabetic"`.
```

Comment:
I don't think string enum like Align should be a special type in the docs. Feel like an overkill to jump to another page for simple string enum.

Reply:
Done